app/components/project-form.js

```diff
@@ -19,6 +20,10 @@ export default class ProjectFormComponent extends Component {
   async save(model) {
     const project = await model.save();
 
+    // the project, when it's created, needs to save all the related models
+    // again, this should happen in the server
```

Comment:
model.save() returns `project`, and then we use project object to save the geometric properties.. why not do that as part of `model.save()`?

Reply:
Hmm I think I understand your question - the `save` method on a model does not implicitly save all related models. I don't know of a way to do this in a single request. My comment here suggest a newly-created project, upon a post to the server, comes back with the geometric properties embedded in the response.